src/main/java/com/rk/rentacarapp/utils/JwtUtil.java

```diff
@@ -0,0 +1,144 @@
+package com.rk.rentacarapp.utils;
+
+import com.rk.rentacarapp.model.User;
+import io.jsonwebtoken.Claims;
+import io.jsonwebtoken.Jwts;
+import io.jsonwebtoken.SignatureAlgorithm;
+import org.springframework.security.core.userdetails.UserDetails;
+import org.springframework.stereotype.Component;
+
+import javax.servlet.http.HttpServletRequest;
+import java.util.Date;
+
+@Component
+public class JwtUtil {
+
+    private String APP_NAME = "Rent a car application";
+
+    public String SECRET = "ourLittleSecret, psst!";
+
+    private int EXPIRES_IN = 180; //Value represents minutes (so.. 3h)
+
+    private String AUTH_HEADER = "Authorization";
+
+    private SignatureAlgorithm SIGNATURE_ALGORITHM = SignatureAlgorithm.HS256;
+
+    // Method for generating new JWT token
+
```

Comment:
Comments above methods should be on the line directly above the method, so in this case line 27

Although the comment in this case seems redundant as the method is pretty descriptive already